static/js/github-configuration.js

```diff
@@ -28,3 +28,13 @@ $('.delete-link').click(function (event) {
     window.close()
   })
 })
+
+$('.logout-link').click(function (event) {
+  event.preventDefault();
+
+	window.open('https://github.com/logout','_blank');
+
+	window.setTimeout(function() {
+		this.close()
+	}, 100)
```

Comment:
If I read this right, you click on the logout link, ignore default behavior, open up a new tab for the github logout (which we don't save the reference to), then set a timeout to close our current window (/github/configuration) and leave the github logout window open?

Reply:
Is it to make sure the window won't stay opened with the old token?

Reply:
The logout window, which renders the logout button, directs to the login page. This stays open as there is no way for us to redirect from this point.

We're closing github/configuration because this page doesn't know that there has been an account logout/login. We need users to go back to the github config page so we pick up the account change.

Would there be a way to do this differently? If we wanted to keep github/configuration open we'd need to make a request to get a new token right?